ci/bugfix-release-pr-body.md

```diff
@@ -0,0 +1,52 @@
+AUTO: merge $RELEASE_BRANCH into $SOURCE_BRANCH
+
+This PR was automatically generated in response to a push to `master`,
+and is a chance to review any changes that will be included in the release
+branch before merging.  Under most circumstances, this PR will probably be
+a formality.  However, there are a few cases where we may need to be some
+extra work to make sure `$RELEASE_BRANCH` contains what it should after the
+merge:
+
+## If this PR causes a trivial merge conflict
+
+Use the github merge conflict resolution editor to resolve the change and
+commit the change to a new branch, *not to master*.
+
+## If this PR causes a nontrivial merge conflict
+
+1. Decline this PR
+2. Make a new bugfix branch off of `master`
+3. Merge `$RELEASE_BRANCH` into the bugfix branch, resolving the conflict.
```

Comment:
I've re-written the PR body/guidance in 511c395 to be more specific about the steps that need to be taken in each case.  I'm suggesting a branch name of `pr-resolution`, since that seemed appropriate.